src/main/java/uk/gov/hmcts/sscs/domain/corecase/CcdCaseResponse.java

```diff
@@ -0,0 +1,156 @@
+package uk.gov.hmcts.sscs.domain.corecase;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Objects;
+
+public class CcdCaseResponse {
+    
+    @JsonProperty(value = "after_submit_callback_response")
+    private AfterSubmitCallbackResponse afterSubmitCallbackResponse;
+
+    @JsonProperty(value = "callback_response_status")
+    private String callbackResponseStatus;
+    
+    @JsonProperty(value = "callback_response_status_code")
+    private String callbackResponseStatusCode;
+
+    @JsonProperty(value = "case_data")
+    private CcdCase caseData;
```

Comment:
Is this just a placeholder? I don't think this is going to work as the names in the `case_data` json do not match with CcdCase. This would involve creating a new deserialiser which converts the Json into the relevant object.

Reply:
yes it is just a place holder. We need to match the names in case_data json with the CcdCase.